src/umpire/tpl/judy/judyL2Array.h

```diff
@@ -217,7 +217,7 @@ class judyL2Array {
         bool removeEntry( JudyKey key ) {
             if( 0 != ( _lastSlot = ( vector ** ) judy_slot( _judyarray, ( const unsigned char * ) &key, _depth * JUDY_key_size ) ) ) {
                 // _lastSlot->~vector(); //for use with placement new
-                // delete _lastSlot;
```

Comment:
Do we know why the comment above was deleting _lastSlot instead of *_lastSlot?  Also, was this leak in the original source as it was brought over or was it introduced after it was added into Umpire?  The reason I ask is because it would be interesting to understand why it was commented out in the first place.

Reply:
I tried to look at the blame but it wasn't very informative. Also as far as I know the judy tpl consists only of `judy.h` and `judy.c`. I wasn't able to find any `judyL2Array` online.

Reply:
When that happens I blame Marty 😄 

The judy implementation can be found here: https://github.com/mpictor/judy-template

Reply:
So this line was uncommented in the original file.  Was in commented out in order to get it to compile?  The last update to the referenced file was 5 years ago.

Reply:
So in the original implementation they have the `delete _lastSlot`
https://github.com/mpictor/judy-template/blob/122e21cb406392d7a3340657baf18cf0a31e370a/src/judyL2Array.h#L210
Which means that it must have been commented out when it was added to Umpire.

Reply:
uh-huh..  Okay..  I guess that the research of this is beyond the scope of this particular check-in.  But I think it would be prudent to sanity check the differences on this file from the original implementation as a separate task.